projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts

```diff
@@ -92,9 +92,21 @@ export class TuiSheetComponent<T> implements TuiSheetRequiredProps<T>, AfterView
     }
 
     ngAfterViewInit(): void {
-        this.el.nativeElement.scrollTop = [...this.stops, this.sheetTop, this.contentTop][
+        const {nativeElement} = this.el;
+        const scrollTop = [...this.stops, this.sheetTop, this.contentTop][
             this.item.initial
         ];
+
+        // Sometimes on iOS it ignores initial scroll, so we try it 5 times over the course of 200ms
```

Comment:
https://taiga-ui--pr4579-sheet-gpy4m57y.web.app/components/sheet#basic


https://github.com/Tinkoff/taiga-ui/assets/35179038/3d599fd2-6734-4dc3-999a-f24080c2cc5b


___
https://taiga-ui.dev/components/sheet#basic

https://github.com/Tinkoff/taiga-ui/assets/35179038/b03f3489-2c27-468a-b25f-5011d6add775

Reply:
iPhone 13 mini
ios 16.4.1

Reply:
Could you debug it by any chance? What are scrollTop values there?

Reply:
> 
> https://github.com/Tinkoff/taiga-ui/assets/35179038/5f97fcb5-b1f7-4bbc-8ab6-2fe8b1051967
> 
> 

Thank you. In your first video it sometimes shows properly, what numbers are there in that case?

Reply:
The same ones(

https://github.com/Tinkoff/taiga-ui/assets/35179038/6966d3cb-e157-450d-ba79-31cc8d51de81

Reply:
@waterplea I think that problem starts here

https://github.com/Tinkoff/taiga-ui/assets/35179038/1c6ee17d-bda2-4ba4-9c88-cf07f6c9d17d

Reply:
I will try to look into it next week.